worlds/akasha.world/webpack.config.cjs

```diff
@@ -13,7 +13,7 @@ module.exports = Object.assign(baseConfig, {
   context: path.resolve(__dirname),
   entry: {
     main: './src/index.ts',
-    'setup.twind': './src/setup.twind.ts'
+    'setup.twind': './src/setup.twind.ts',
```

Comment:
I'm not sure if the trailling comma is ok in cjs files

Reply:
is just my editor fixing things... reverting this